pkgs/by-name/cl/cli-tips/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  bash,
+  makeWrapper,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "cli-tips";
+  version = "0-unstable-2024-11-14";
+
+  src = fetchFromGitHub {
+    owner = "cli-stuff";
+    repo = "cli-tips";
+    rev = "ebc191a54be7e39accd1948c3de8aded438d0495";
+    sha256 = "sha256-KELgatdL+2M5ktuAHEljIEJ9wqP578dp5tYWYPpP3bg=";
+  };
+
+  buildInputs = [ bash ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  configurePhase = ''
+    runHook preConfigure
+
+    mkdir -p $out/bin $out/share
+    cp cli-tips.sh $out/bin/cli-tips
+    cp -r translations $out/share
+    chmod +x $out/bin/cli-tips
+    rm Makefile # nix tries to execute make in the buildPhase
+
+    runHook postConfigure
+  '';
+
+  postInstall = ''
+    wrapProgram $out/bin/cli-tips \
+      --set TIPS_FOLDER "$out/share/translations"
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "CLI tool that provides useful tips and commands for Linux users";
+    homepage = "https://github.com/cli-stuff/cli-tips";
+    license = licenses.unlicense;
+    platforms = platforms.all;
```

Upstream only claims it works under these Linux/macOS/Windows
```suggestion
    platforms = with lib.platforms; linux ++ darwin ++ windows;
```